examples/nextjs/README.md

```diff
@@ -0,0 +1,133 @@
+# Use with Next.js (with new Next.js compiler)
+
+This example demonstrates how to use `jest-preview` with `Next.js`, bootstrapped using `create-next-app`.
+
+⚠️ This example is only meant for Next.js apps using the new [Next.js compiler](https://nextjs.org/docs/advanced-features/compiler) (also called Rust compiler), available from Next.js 12.0.0 onward. If you use Next.js version 11 or below, or opt out of the Next.js compiler, please see the `nextjs-babel` example.
+
+🚧 Next.js's `<Image />` component isn't working with Jest Preview yet.
+
+## Install Jest
+
+Install Jest and your favorite testing libraries, if you haven't already done so. Refer to [Next.js's official document](https://nextjs.org/docs/testing). This example uses `react-testing-library`.
+
+## Install & configure Jest Preview
+
+Install Jest Preview
+
+```bash
+npm install --save-dev jest-preview
+```
+
+Enable Jest Preview inside `jest.config.js`. Note that `enableJestPreview` is an async function that accept another async function as its parameter. The Jest config object returned by `enableJestPreview` shouldn't be modified any further.
```

Comment:
I don't think we need to clarify how Jest works with config here. Just give them the example is sufficient.

Reply:
Okay